Applications/SimpleShot/layers/centering.cpp

```diff
@@ -69,15 +62,18 @@ void CenteringLayer::read(std::ifstream &file) {
   mean_feature_vector.read(f);
 }
 
-void CenteringLayer::forwarding(bool training) {
-  std::cout << net_input[0]->getVariableRef().getDim();
-  std::cout << net_hidden[0]->getVariableRef().getDim();
+void CenteringLayer::forwarding(nntrainer::RunLayerContext &context,
+                                bool training) {
+  auto &hidden_ = context.getOutput(SINGLE_INOUT_IDX);
+  auto &input_ = context.getInput(SINGLE_INOUT_IDX);
+
+  std::cout << input_.getDim();
+  std::cout << hidden_.getDim();
```

Comment:
It is just for the debugging?

Reply:
this existed in the original code, but removed now.